Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -27,21 +27,6 @@ open class NavigationMapView: UIView {
     // TODO: Mapbox Maps should provide the ability to set custom `PreferredFPS` value.
     public var minimumFramesPerSecond = PreferredFPS.normal
     
-    /**
-     Returns the altitude that the map camera initally defaults to.
-     */
-    public var defaultAltitude: CLLocationDistance = 1000.0
-    
-    /**
-     Returns the altitude the map conditionally zooms out to when user is on a motorway, and the maneuver length is sufficently long.
-     */
-    public var zoomedOutMotorwayAltitude: CLLocationDistance = 2000.0
-    
-    /**
-     Returns the threshold for what the map considers a "long-enough" maneuver distance to trigger a zoom-out when the user enters a motorway.
-     */
-    public var longManeuverDistance: CLLocationDistance = 1000.0
```

Comment:
Since new camera related changes will automatically adapt to road conditions such properties are no longer used.

Reply:
In initial version I'd rather keep public APIs as simple as possible, and then in future iterations add only properties in case if those are really required. Also in case if users would like custom camera behavior it'd be possible to conform to `ViewportDataSource` directly. From the ones listed here `defaultAltitude` (adapted to zoom level) can be exposed.